pkg/process/exec.go

```diff
@@ -142,10 +144,11 @@ func (e *execProcess) Kill(ctx context.Context, sig uint32, _ bool) error {
 
 func (e *execProcess) kill(ctx context.Context, sig uint32, _ bool) error {
 	pid := e.pid.get()
-	if pid != 0 {
-		if err := unix.Kill(pid, syscall.Signal(sig)); err != nil {
-			return errors.Wrapf(checkKillError(err), "exec kill error")
-		}
+	if pid <= 0 {
```

Comment:
Why change the exec kill behavior, return error here

Reply:
this was just because it was based on a not yet merged PR.